src/v3/src/components/LaunchAuthenticatorButton/LaunchAuthenticatorButton.tsx

```diff
@@ -79,34 +79,19 @@ const LaunchAuthenticatorButton: UISchemaElementComponent<{
   return (
     <OdyButton
       variant="secondary"
-      fullWidth
+      isFullWidth
       onClick={handleClick}
-      ref={focusRef}
-      // Fixes text overflow
-      sx={{ display: 'flex', whiteSpace: 'normal' }}
-    >
-      <Box
-        display="flex"
-        alignItems="center"
-        justifyContent="center"
-      >
-        <Box
-          sx={{
-            marginInlineEnd: '5px',
-            // keep the icon from stretching the button vertically
-            marginBlockEnd: '-3px',
-          }}
-        >
```

Comment:
Have you tested whether this fix was reintroduced by removing these styles?

Reply:
Yes. The fix is not needed after `styleOverrides` for `MuiButton`: https://github.com/okta/okta-signin-widget/pull/3502/files/7c133eb6bd7625bf0f82cbef3891281d3328ef3f#diff-18b2c20ca070dc0820e0e72efb48fb243e99865d2307d4b3f73353fd05debe10R126-R128